src/components/SignUp/Landing.tsx

```diff
@@ -175,6 +176,15 @@ const Landing: React.FC<{
           </Row>
         </Col>
       </Row>
+      <Row>
+        <TaglineTwo>
+          {'Already a member?'}
+          <a href="/login" style={{ color: colors.orange }}>
```

Comment:
Instead of using `style` attributes, you want to create a styled component with that style. Something along the lines of:
```
const OrangeLink = styled.a`
  color: ${colors.orange}
`;
```

And, since this may be a more global style, we may want to create a generic component for it somewhere else that we can import here and use.